hashicorp/dev-portal
# style: cleanup open-api-docs typography

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksapi-docs-accordion-hashicorp.vercel.app/boundary/api-docs/group-service) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1204129860249685) 🎟️
- [Figma](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=12849%3A162751&t=LlSZjMQrZ5lFg6wj-0)

## 🗒️ What

- Updates typography to[ match this spec](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=12849%3A162751&t=LlSZjMQrZ5lFg6wj-0).
- Removes old mktg-global-styles token references. 

### Dark theme screenshot from local dev
<img width="1255" alt="Screenshot 2023-03-14 at 1 11 56 PM" src="https://user-images.githubusercontent.com/36613477/225125255-308d8961-9e5c-45ab-88d4-b2de3ad1fb03.png">


## 📸 Design Screenshots

<img width="722" alt="Screenshot 2023-03-14 at 1 10 12 PM" src="https://user-images.githubusercontent.com/36613477/225124888-400b52f0-4ac7-454d-97ec-d948d85a8349.png">


## 🧪 Testing

- Visit [Preview link](https://dev-portal-git-ksapi-docs-accordion-hashicorp.vercel.app/boundary/api-docs), [hcp packer](https://dev-portal-git-ksapi-docs-accordion-hashicorp.vercel.app/hcp/api-docs/packer), and [waypoint api docs,](https://dev-portal-git-ksapi-docs-accordion-hashicorp.vercel.app/waypoint/api-docs) validate that the type colors and styles match the spec for both dark and light mode.
- Ensure the open api page disclosure [accordion on swingset](https://dev-portal-git-ksapi-docs-accordion-hashicorp.vercel.app/swingset/components/openapipage) isn't affected by the new prop for light mode.

## 💭 Anything else?

We will [follow up](https://app.asana.com/0/1199634971449915/1204167616054159) with an override for the accordion background in dark mode to be surface-faint to match the card override.
